### DISCUSSION TITLE
nuxt script property on nuxt.config.js not working

### DISCUSSION THREAD

**Discussion by ezioadf2** [nuxt/nuxt#17276] [Questions] [answered]
hi 

in nuxt.confing.js:

`head: {
        script: [
            // <script src="https://myawesome-lib.js"></script>
            { src: 'https://accounts.google.com/gsi/client' }
          ],
    },
    script: [
        "https://accounts.google.com/gsi/client",
    ]`

i try to add this script to my website  but when i inspect page , on resource tab there is no file from google.com and function of the script not working
what i can do?

**Comment by Flanker72** [comment, marked as answer]
Seems you have incorrect `script` tag definition.
Please, try this format:
```ts
import { defineNuxtConfig } from "nuxt";

export default defineNuxtConfig({
  app: {
    head: {
      script: [{ src: "https://accounts.google.com/gsi/client" }],
    },
  },
});
```
As from the description, `Each item in the array maps to a newly-created <script> element, where object properties map to attributes.`. So, if you need to pass any additional attributes like `async`, `type`, `crossorigin` etc - you may define them as properties of the given object as well like the `src`.
